### ISSUE TITLE
Update help text and labelling surrounding headings

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#8293]
# Problem

>One item I've noticed that may help with the ever-present issue of trying to get folks to use correct heading levels: the language in blocks like the Text Area or others could be adjusted to say "Heading level" instead of "Heading size" and maybe link to WCAG guidelines about heading levels. People like to consider h1,h2,h3, et al as just  different formatting "sizes" and probably should be guided to the proper use of the heading levels instead. - @quamsta https://iowaweb.slack.com/archives/C0164FNN1PV/p1732720916916359?thread_ts=1732716599.744289&cid=C0164FNN1PV

![Screenshot 2024-11-27 at 9 18 39 AM](https://github.com/user-attachments/assets/ce894b1b-4794-4134-b23a-046ea5c056f2)
